test/unit/collection-view/collection-view-children.spec.js

```diff
@@ -204,6 +204,78 @@ describe('CollectionView Children', function() {
       this.sinon.spy(myCollectionView, 'sort');
     });
 
+    describe('when called with preventRender option', function() {
+
+      beforeEach(function() {
+        myCollectionView.viewComparator = false;
+        myCollectionView.addChildView(addView, { preventRender: true });
+      });
+
+      it('should return the added view', function() {
+        expect(myCollectionView.addChildView).to.have.returned(addView);
+      });
+
+      it('should add to the children container', function() {
+        expect(myCollectionView.children._add)
+          .to.have.been.calledOnce.and.calledWith(addView);
+      });
+
+      it('should not call sort', function() {
+        expect(myCollectionView.sort)
+          .to.be.not.called;
+      });
+
+      it('should not trigger "before:render:children"', function() {
+        expect(myCollectionView.onBeforeRenderChildren)
+          .to.be.not.called;
+      });
+
+      it('should not trigger "render:children"', function() {
+        expect(myCollectionView.onRenderChildren)
+          .to.be.not.called;
+      });
+
+      it('should trigger "add:child"', function() {
+        expect(myCollectionView.onAddChild)
+          .to.be.calledOnce.and.calledWith(myCollectionView, addView);
+      });
+
+      it('should trigger "before:add:child"', function() {
+        expect(myCollectionView.onBeforeAddChild)
+          .to.be.calledOnce.and.calledWith(myCollectionView, addView);
+      });
+
+    });
+
+    describe('when called with an index in options', function() {
+      const addIndex = 1;
+      beforeEach(function() {
+        myCollectionView.viewComparator = false;
+        myCollectionView.addChildView(addView, 0, { preventRender: true, index: 1 });
+      });
+
+      it('should add to the children container at the index from options', function() {
+        expect(myCollectionView.children._add)
+          .to.have.been.calledOnce.and.calledWith(addView, addIndex);
+      });
+
+    });
+
+    describe('when called without preventRender after preventReder calls', function() {
+      const addViewIndex = 1;
+      beforeEach(function() {
+        const addView2 = new View({ template: _.noop });
+        myCollectionView.viewComparator = false;
```

Comment:
why is `viewComparator` set here?

Reply:
i've done requested test changes (not commited yet)
should i add additional tests or its enough?